openframe-data-mongo-sync/src/main/java/com/openframe/data/repository/device/.CustomMachineRepositoryImpl.md

```
@@ -40,4 +40,6 @@ List<Machine> nextPage = repository.findMachinesWithCursor(
 );
 ```
 
-> **Note:** Cursor-based pagination avoids offset drift on live data. When paginating on a non-unique sort field (e.g., `status`, `lastSeen`), ties are broken by `_id` to ensure stable, gap-free page traversal.
\ No newline at end of file
+> **Note:** Cursor-based pagination avoids offset drift on live data. When paginating on a non-unique sort field (e.g., `status`, `lastSeen`), ties are broken by `_id` to ensure stable, gap-free page traversal.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)